src/components/Footer/Footer.tsx

```diff
@@ -23,39 +35,40 @@ export const Footer = ({ screenWidth = "desktop" }: Props) => {
           isMobile ? "flex-col !gap-[26px]" : "flex-row"
         }`}
       >
-        <div
-          className={`flex flex-col gap-[40px] sm:gap-[26px] ${
-            isMobile ? "gap-[26px]" : "gap-[40px]"
-          }`}
-        >
-          <div className="flex flex-col gap-[24px] ">
-            <p
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-            >
-              Expert Event Solutions
-            </p>
-            <a
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-              onClick={() => {}}
-            >
-              260 Daytona Blvd, Daytona <br /> Beach, Florida 32114, United{" "}
-              <br /> States
-            </a>
-            <a
-              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
-                isMobile ? "text-center" : "text-left"
-              }`}
-              onClick={() => {}}
-              href="mailto:Info@experteventsolutions.com"
-            >
-              Info@experteventsolutions.com
-            </a>
+        {companyContactInfo && companyContactInfo[0] ? (
+          <div
+            className={`flex flex-col gap-[40px] sm:gap-[26px] ${
+              isMobile ? "gap-[26px]" : "gap-[40px]"
+            }`}
+          >
+            <div className="flex flex-col gap-[24px] ">
+              <p
+                className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
+                  isMobile ? "text-center" : "text-left"
+                }`}
+              >
+                {companyContactInfo ? companyContactInfo[0]?.companyName : ""}
+              </p>
+              <a
+                className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer w-[271px] ${
+                  isMobile ? "text-center" : "text-left"
+                }`}
+                onClick={() => {}}
+              >
+                {companyContactInfo ? companyContactInfo[0]?.address : ""}
```

Comment:
The address seems to not render correctly for me now that it is connected to the backend. 
And that's because on the backend, the address field isn't asking for a full address. 
This can be mitigated by either asking for the full address from the client on the backend, or adding more field (like zip code and state) that are missing from the backend and then concatenating them to make the full address.

Reply:
I am not sure how you're address renders, but when I run my backend this is what the footer looks like for me.
<img width="854" alt="Screenshot 2024-09-21 at 11 01 36" src="https://github.com/user-attachments/assets/10c697f7-c380-46c4-af05-2d2770d6a970">
Also I've attached a screenshot of how I have set up the data in keystone 
<img width="1230" alt="image" src="https://github.com/user-attachments/assets/4f3cb8f8-d8bf-40b6-bed8-9b00bbc09f64">

Reply:
So I am not sure what yours looks like when you are running the backend and the frontend together, but this is what mine looks like when I run `npm run dev` on the backend and frontend together. 
<img width="1652" alt="image" src="https://github.com/user-attachments/assets/94b2e21c-bf2a-43cb-8a80-38b128c35ee4">
 Also here is what my backend data looks like: 
<img width="1652" alt="image" src="https://github.com/user-attachments/assets/3ea4d398-5f5e-48a0-ab32-2465ddc335bb">
 The issues you're encountering could have something to do with the way you have structured this list on your end.